src/kernel/include/shared/cos_types.h

```diff
@@ -131,6 +132,7 @@ typedef enum {
 typedef enum {
 	CAP_FREE = 0,
 	CAP_SINV,       /* synchronous communication -- invoke */
+	CAP_SINVFLT,    /* synchronous communication of fault handler -- invoke*/
```

Comment:
Because it is really not an "INVOCATION" from USER-LEVEL, I think it can just be named `CAP_FAULT` or `CAP_FLT`